qa/rpc-tests/fundrawtransaction.py

```diff
@@ -316,7 +316,7 @@ def run_test(self):
         rawtx   = self.nodes[2].createrawtransaction(inputs, outputs)
         dec_tx  = self.nodes[2].decoderawtransaction(rawtx)
 
-        assert_raises_jsonrpc(-4, "Insufficient funds", self.nodes[2].fundrawtransaction, rawtx)
+        assert_raises_jsonrpc(-4, "Coin Selection Failed", self.nodes[2].fundrawtransaction, rawtx)
```

Comment:
I think "Insufficient funds" is correct. If the user does not have enough liquidity to not do a long chain, then he does not have enough funds.